archlinuxcn/wine-wechat/wine-wechat.desktop

```diff
@@ -11,3 +11,19 @@ Terminal=0
 Type=Application
 Categories=Network;InstantMessaging;
 Keywords=wx;wechat;weixin
+StartupWMClass=wechat
+Actions=Configure;Directory;
+
+[Desktop Action Configure]
+Exec=env WINEDEBUG=-all wechat --config
+Name=Configure Wine
```

Comment:
This message is inaccurate. It's for configuring "Wine Wechat".

Reply:
Sure, but technically it should be "configure wine (winecfg) for wechat", not "configure wechat", which sounds like configuring wechat itself (e.g. opening its settings). But I agree with you. Please suggest how to phrase this more concisely.